frontend/packages/ceph-storage-plugin/src/utils/alert-action-path.tsx

```diff
@@ -1,13 +1,8 @@
-import { Alert } from '@console/internal/components/monitoring/types';
-import { history } from '@console/internal/components/utils';
 import { k8sGet } from '@console/internal/module/k8s';
 import { CEPH_STORAGE_NAMESPACE } from '../constants';
 import { OCSServiceModel } from '../models';
 
-export const getAlertActionPath = (alertData: Alert) =>
-  history.push(
-    `/k8s/cluster/nodes/${alertData.labels.host}/disks?node-disk-name=${alertData.labels.device}`,
-  );
+export const getAlertActionPath = () => window.open('https://access.redhat.com/solutions/5194851');
```

Comment:
can we name it as getDiskAlertActionPath

Reply:
no opinion

Reply:
Also, we need to make sure this URL wont change when any edit is happneing

Reply:
wont change surely/